lib/ansible/modules/network/netvisor/pn_vrouter.py

```diff
@@ -411,6 +411,7 @@ def main():
     run_cli(module, cli)
 
 # AnsibleModule boilerplate
+
 from ansible.module_utils.basic import AnsibleModule
```

Comment:
I'm okay with you moving these sorts of imports up with the other imports near the top of the file (as you do in some of the other files)